elements/pl-xss-safe/pl-xss-safe.mustache

```diff
@@ -0,0 +1,14 @@
+<script>
+(function() {
+    let code = {{{quoted_code}}};
+    let language = "{{language}}";
+    if (language == "markdown") {
+        let renderer = new showdown.Converter({
```

Comment:
Where does `showdown` get included on the page? Does this need to be listed as a client-side dependency?

Reply:
This was an oversight on my part, these come from dependencies in `pl-file-editor` that should have been added as dependencies here as well. I just hadn't tested it in a question without this element. I'll review it.

Reply:
Makes sense!